### PR TITLE
[CS] Code Style fixes for administrator/components/com_fields/

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style fixes

### Summary of Changes
- No blank line found after control structure
- Please consider an empty line before the if statement;
- Expected 1 space after "!="; 0 found
- Expected 1 space after FUNCTION keyword; 0 found
- No blank line found after control structure
- Closing parenthesis of a multi-line function call must be on a line by itself
- Object operator not indented correctly;
- Please consider an empty line before the foreach statement;
- Expected 0 spaces before opening parenthesis; 1 found
- Multi-line function call not indented correctly;
- Whitespace found at end of line

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to impliment fixable codestyle

### Documentation Changes Required
none

### CODE REVIEW

**Review by laoneo — APPROVED** [review]

File: administrator/components/com_fields/helpers/fields.php [hunk 1]
```diff
@@ -146,7 +148,7 @@ public static function getFields($context, $item = null, $prepareValue = false,
 			}
 
 			$fieldIds = array_map(
-				function($f)
+				function ($f)
```
Comment (photodude) [inline administrator/components/com_fields/helpers/fields.php:151]: @mbabker @wilsonge any thoughts on this change? Something about it seems wrong to me.

Comment (zero-24) [reply administrator/components/com_fields/helpers/fields.php:151]: Why? The rule is to have a space between function and the opening ( or does that produce syntax errors here?

Comment (photodude) [reply administrator/components/com_fields/helpers/fields.php:151]: @zero-24 I thought the rule was "Functions should be called with no spaces between the function name and the opening parenthesis" whereas "Function definitions start on a new line with no spaces between the function name and the opening parenthesis and the opening and closing braces are also placed on new lines."

Comment (mbabker) [reply administrator/components/com_fields/helpers/fields.php:151]: Closures like this generally have the space in between.  Method signatures don't have the space.

Comment (photodude) [reply administrator/components/com_fields/helpers/fields.php:151]: We should clarify the manual on that.

Comment (photodude) [reply administrator/components/com_fields/helpers/fields.php:151]: PR for clarifying the manual
https://github.com/joomla/coding-standards/pull/194

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e20e96da7acc3858efe545b9c5906318092ad6bd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16914">issues.joomla.org/tracker/joomla-cms/16914</a>.</sub>

**Comment by brianteeman** [comment]
@photodude can you fix the conflicts?

**Comment by zero-24** [comment]
are you sure that you can remove the new lines? As they come from the custom fields commit.? https://github.com/photodude/joomla-cms/blame/staging/administrator/components/com_fields/models/fields/type.php#L76

**Comment by photodude** [comment]
@zero-24 ugh, Seems something went wrong in fixing the merge conflicts here. I guess that's what I get for trying to get it done before vacation.

**Comment by photodude** [comment]
@zero-24 I just looked at staging... those lines are not there. https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_fields/models/fields/type.php#L51-L83

File: administrator/components/com_fields/models/fields/type.php [hunk 2]
```diff
@@ -78,7 +78,8 @@ function typeHasChanged(element){
 				jQuery('input[name=task]').val('field.reload');
 				element.form.submit();
 			}
-		");
+		"
+		);
```
Comment (wilsonge) [inline administrator/components/com_fields/models/fields/type.php:82]: These new lines seem a bit weird

Comment (photodude) [reply administrator/components/com_fields/models/fields/type.php:82]: I believe this is due to the rule which requires "Closing parenthesis of a multi-line function call must be on a line by itself" or a similar rule

**Comment by zero-24** [comment]
ah Sorry @photodude i got confused because i have looked in your staging branch and not the joomla one ;) 

I have also reboot travis so this should pass in a few minutes too.

**Comment by photodude** [comment]
Thank for restarting Travis @zero-24

File: administrator/components/com_fields/models/fields/type.php (hunk 2 above)
Comment (photodude) [reply administrator/components/com_fields/models/fields/type.php:82]: I also agree it's a little odd, I think the old way we could "fix it" is manually adjusting where the quotations are. (of course I'm trying to keep all of these to just the automated tests so we can focus on verifying the use of the automated fixers)

**Comment by wilsonge** [comment]
Merging for now. But I think https://github.com/joomla/joomla-cms/pull/16914/files/61af4a0cc311194c31e613e3175c9768ad767b3c#diff-f7da601c0a51808627cf42d006fbe02a needs a bit of thought as although it is indeed compliant with the sniffer seems a bit odd. Maybe open it on the code style repo

**Comment by photodude** [comment]
Definitely an issue for the code style repo

**Pull request merged**